backend/user-service/.env.sample

```diff
@@ -0,0 +1,2 @@
+MONGO_URI="maxhasntsendyet"
```

Comment:
real

Reply:
i think should rename to `USER_SERVICE_MONGODB_URI`

Reply:
Ok fixed.